src/transport/SecureSessionMgr.cpp

```diff
@@ -82,6 +82,21 @@ CHIP_ERROR SecureSessionMgr::Init(NodeId localNodeId, System::Layer * systemLaye
     return err;
 }
 
+Transport::Type SecureSessionMgr::GetTransportType(NodeId peerNodeId)
+{
+    PeerConnectionState * state = mPeerConnections.FindPeerConnectionState(peerNodeId, nullptr);
```

Comment:
I assume this is safe because we can't have both a TCP and a UDP connection open to the same peer node at the same time, so there is only one transport type to find?  Might be worth documenting the invariants around this somewhere..

Reply:
Yes, CHIP can only have one connection to each peer node, but multiple exchange over this connection. See documentation in the header file.